Modules/socketmodule.h

```diff
@@ -101,39 +101,39 @@ typedef int socklen_t;
 #include <sys/kern_control.h>
 #endif
 
-#ifdef HAVE_SOCKADDR_ALG
-#include <linux/if_alg.h>
-#ifndef AF_ALG
-#define AF_ALG 38
-#endif
-#ifndef SOL_ALG
-#define SOL_ALG 279
-#endif
-
 #ifdef HAVE_LINUX_VM_SOCKETS_H
 # include <linux/vm_sockets.h>
 #else
 # undef AF_VSOCK
 #endif
 
+#ifdef HAVE_SOCKADDR_ALG
```

Comment:
It would help to have a space after this line to make it symmetric with the line before the `#endif`.

Reply:
This makes sense, amended my pull request.